plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -216,6 +216,7 @@
 import static io.trino.plugin.iceberg.IcebergTableProperties.getPartitioning;
```

Comment:
Follow the guideline for commit comments for https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#commits-and-pull-requests